<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UASMultiControl</class>
 <widget class="QWidget" name="UASMultiControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>201</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#colorIcon {}

QWidget {
background-color: #050508;
color: #DDDDDF;
background-clip: border;
font-size: 11px;
}

QLabel {
background-color: transparent;
}

QLabel#nameLabel {
	font: bold 16px;
	color: #3C7B9E;
}

QLabel#modeLabel {
	font: 12px;
}

QLabel#stateLabel {
	font: 12px;
	color: #3C7B9E;
}

QLabel#navLabel {
	font: 12px;
}

QLabel#positionLabel {
	font: 8px;
}

QLabel#timeElapsedLabel {
	font: 8px;
}

QLabel#groundDistanceLabel {
	font: 8px;
}

QLabel#speedLabel {
	font: 8px;
}

QLabel#timeRemainingLabel {
	font: 8px;
}

QLabel#waypointLabel {
	font: 22px;
}

QGroupBox {
	border: 1px solid #4A4A4F;
	border-radius: 10px;
	padding: 0px 0px 0px 0px;
	margin: 0px;
}

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     margin: 0 3px 0px 3px;
     padding: 0 3px 0px 0px;
     font: bold 8px;
 }

QGroupBox#heartbeatIcon {
	background-color: red;
}

QToolButton#typeButton {
	font-weight: bold;
	font-size: 12px;
	border: 0px solid #999999;
	border-radius: 5px;
	min-width:42px;
	max-width: 42px;
	min-height: 42px;
	max-height: 42px;
	padding: 0px;
	margin: 0px;
	background-color: none;
}

QPushButton {
	font-weight: bold;
	font-size: 12px;
	border: 1px solid #999999;
	border-radius: 8px;
	min-width: 20px;
	max-width: 60px;
	min-height: 16px;
	max-height: 16px;
	padding: 2px;
	spacing: 10px;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #777777, stop: 1 #555555);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #444444, stop: 1 #555555);
}

QPushButton#abortButton {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                             stop:0 #ffee01, stop:1 #ae8f00);
}

QPushButton:pressed#abortButton {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                             stop:0 #bbaa00, stop:1 #a05b00);
}

QPushButton#killButton {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                             stop:0 #ffb917, stop:1 #b37300);
}

QPushButton:pressed#killButton {
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                             stop:0 #bb8500, stop:1 #903000);
}


QProgressBar:horizontal {
	border: 1px solid #4A4A4F;
	border-radius: 4px;
	text-align: center;
	padding: 2px;
	color: #111111;
	background-color: #111118;
}

QProgressBar:vertical {
	border: 1px solid #4A4A4F;
	border-radius: 4px;
	text-align: center;
	font-size: 7px;
	padding: 2px;
	color: #DDDDDF;
    min-width: 16px;
	background-color: #111118;
}

QProgressBar:horizontal {
	height: 10px;
}

QProgressBar:horizontal QLabel {
	font-size: 9px;
	color: #111111;
}

QProgressBar:vertical QLabel {
	font-size: 7px;
}

QProgressBar:vertical {
	width: 14px;
}

QProgressBar::chunk {
	background-color: #656565;
}

QProgressBar::chunk#batteryBar {
	background-color: green;
}

QProgressBar::chunk#speedBar {
	background-color: yellow;
}

QProgressBar::chunk#thrustBar {
	background-color: orange;
}

QToolTip {
        background-color: #090909;
        border: 1px solid #379AC3;
	border-radius: 3px;
    color: #DDDDDF;
}

QMenu {
    border: 1px solid #379AC3;
background-color: #050508;
color: #DDDDDF;
background-clip: border;
font-size: 11px;
}

QMenu::separator {
     height: 1px;
     background: #379AC3;
     margin-top: 8px;
     margin-bottom: 4px;
     margin-left: 5px;
     margin-right: 5px;
 }</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="uasViewFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>300</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="armButton">
          <property name="text">
           <string>ARM_ALL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disarmButton">
          <property name="text">
           <string>DISA_ALL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="armMessageLabel">
          <property name="text">
           <string>no arm message</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>LiftOff   Halt   Continue   Land   ShutDown   Abort    Kill</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,1,1,1">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="liftoffButton">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>22</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Liftoff / Launch</string>
          </property>
          <property name="statusTip">
           <string>Liftoff / Launch</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../qgroundcontrol.qrc">
            <normaloff>:/files/images/control/launch.svg</normaloff>:/files/images/control/launch.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="haltButton">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>22</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Loiter / Wait at current position</string>
          </property>
          <property name="statusTip">
           <string>Loiter / Wait at current position</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../qgroundcontrol.qrc">
            <normaloff>:/files/images/actions/media-playback-pause.svg</normaloff>:/files/images/actions/media-playback-pause.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="continueButton">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>22</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Continue flightplan</string>
          </property>
          <property name="statusTip">
           <string>Continue flightplan</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../qgroundcontrol.qrc">
            <normaloff>:/files/images/actions/media-playback-start.svg</normaloff>:/files/images/actions/media-playback-start.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="landButton">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>22</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Fly straight to landing location</string>
          </property>
          <property name="statusTip">
           <string>Fly straight to landing location</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../qgroundcontrol.qrc">
            <normaloff>:/files/images/control/land.svg</normaloff>:/files/images/control/land.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shutdownButton">
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>22</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Only in standby mode: Power off system</string>
          </property>
          <property name="statusTip">
           <string>Only in standby mode: Power off system</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../qgroundcontrol.qrc">
            <normaloff>:/files/images/actions/system-log-out.svg</normaloff>:/files/images/actions/system-log-out.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="abortButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>22</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Emergency land system at closest possible site</string>
          </property>
          <property name="statusTip">
           <string>Emergency land system at closest possible site</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../qgroundcontrol.qrc">
            <normaloff>:/files/images/actions/media-playback-stop.svg</normaloff>:/files/images/actions/media-playback-stop.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="killButton">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>22</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Kill immediately all onboard power</string>
          </property>
          <property name="statusTip">
           <string>Kill immediately all onboard power</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../qgroundcontrol.qrc">
            <normaloff>:/files/images/actions/process-stop.svg</normaloff>:/files/images/actions/process-stop.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QGridLayout" name="shortcutGroupBox">
        <item row="1" column="1">
         <widget class="QPushButton" name="acroModeButton">
          <property name="text">
           <string>Acro</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="altHoldModeButton">
          <property name="text">
           <string>Alt Hold</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="landModeButton">
          <property name="text">
           <string>Land</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="posHoldModeButton">
          <property name="text">
           <string>Pos Hold</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="rtlModeButton">
          <property name="text">
           <string>RTL</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="steerModeButton">
          <property name="text">
           <string>Steer</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="holdModeButton">
          <property name="text">
           <string>Hold</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="stabilizeModeButton">
          <property name="text">
           <string>Stabilize</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="autoModeButton">
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="loiterModeButton">
          <property name="text">
           <string>Loiter</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="learnModeButton">
          <property name="text">
           <string>Learn</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QComboBox" name="modeComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qgroundcontrol.qrc"/>
 </resources>
 <connections/>
</ui>
